### DISCUSSION TITLE
Monthly ESMValTool meeting December 2021

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#5] [Meetings]
The next monthly ESMValTool meeting will be in December. The result of the poll is that it will be on

Thursday, December 2 at 15:00 CET (14:00 GMT).

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

@ESMValGroup/esmvaltool-developmentteam

Format and agenda

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we can decide to talk about specific topics with the entire group or make smaller breakout groups to continue the discussion there.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the November meeting can be found here: https://github.com/ESMValGroup/Community/discussions/4#discussioncomment-1595111

**Comment by bouweandela** [comment]
In case you're wondering why there are so few options in the doodle: I'm only working Thursdays and Fridays at the moment. If there is a preference for another day, I would need someone else to take over chairing the meeting. Please comment here if you would like to have the meeting on another day of the week.

**Comment by rswamina** [comment]
I will not be able to attend this meeting. Apologies. Over December and January, I plan to work on updating the ESMValTool tutorial (with others in the user enaggement team) to be compatible with the latest release of ESMValTool (v2.4).

**Comment by ehogan** [comment]
The Zoom link tells me the meeting has not started. Is the link above correct?

**Reply by remi-kazeroni** [reply]
I have the same issue that the meeting hasn't started

**Comment by katjaweigel** [comment]
Tina and me are waiting as well?

**Comment by bettina-gier** [comment]
Should we give it till quarter past and if there's no change, we make a new zoom link and proceed without Bouwe? Who knows if something came up.

**Comment by bettina-gier** [comment]
https://uni-bremen.zoom.us/j/96936122040?pwd=dVV0b1F1T2hBMld1SkxtK2pnU1V6Zz09  Let's try this!

**Comment by bettina-gier** [comment]
Minutes for December Meeting (we were quite fas, probably too soon after the workshop ;)):

Personal updates:
@valeriupredoi general maintenance, helped with tutorial github action testing
@ehogan Converting autoassess metrics into ESMValTool, PR up for radiation; Working on Tutorial for MetOffice staff; Working on testing recipes
@remi-kazeroni Working with Javi on automatic data download PR
@sloosvel is waiting for a review from Bouwe
Catherine Hardacre attended ESMValTool workshop

Further topics:
Multi-Model Means merge cubes having problems with attributes like differing long_names:
Emma had discussion with @bjlittle, iris might implement a more lenient cube merge please upvote this issue on the iris github!: https://github.com/SciTools/iris/issues/4446

Otherwise have a Merry Christmas and a Happy New Year everyone!

**Comment by bouweandela** [comment]
My apologies for not being able to host the meeting this time and for not letting you know. My wife and oldest daughter were ill on the day of the meeting and I completely forgot about it. I'm happy to see that you still had a good meeting without me.
